lib/jormungandr/src/Cardano/Environment.hs

```diff
@@ -0,0 +1,166 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- This module contains static configuration parameters. Rather than providing
+-- and carrying around a configuration file through the application, we resolve
+-- configuration data at runtime using the available environment.
+--
+-- This gives us a flexible and portable approach to software configuration, and
+-- remove some pain from the development perspective. Prior to starting, the
+-- wallet is expected to have a few configuration parameter available. One may
+-- rely on a `.env` file to bundle configuration settings together for a given
+-- target environment.
+
+module Cardano.Environment
```

Comment:
I wasn't sure moving `lib/{jormungandr, http-bridge}/Cardano/Environment.hs` to `cardano-wallet-core` was desired, so I kept the duplication for now.

Reply:
Well, network discrimination is different on `Jörmungandr`, and probably will be too with Shelley Haskell. There are actually only too network (testnet or mainnet) and it's encoded on a single byte (Cf the new address format). Perhaps it'd make sense to rename both modules to `Cardano.Environment.{Jormungandr,HttpBridge}`.

Reply:
Yet, we could factor out in `core` the "Internal" bits with the `ErrMissingOrInvalidEnvVar` and `unsafeLookupEnv` since this is fairly network-agnostic.